src-built-in/components/processMonitor/processMonitor.css

```diff
@@ -78,6 +78,7 @@ body {
 .value-cell {
 	display:flex;
 	justify-content: flex-end;
+	width: 20%;
```

Comment:
@carterbs try to start unrelated projects from master. Having processMonitor stuff in this branch could cause any number of merge problems.